ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/PreviousOrganisationCollectionItem.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.ccd.sdk.type;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.ccd.sdk.api.ComplexType;
+
+@NoArgsConstructor
+@Builder
+@Data
+@ComplexType(name = "PreviousOrganisationCollectionItem", generate = false)
+public class PreviousOrganisationCollectionItem {
+
+  @JsonProperty("id")
+  private String id;
+
+  @JsonProperty("value")
+  private PreviousOrganisation value;
+
+  @JsonCreator
```

Comment:
this boilerplate should be replaceable with an @AllArgsConstructor lombok annotation on the class

Reply:
Replaced with Lombok annotations